compiler/rustc_hir_typeck/src/op.rs

```diff
@@ -865,11 +865,12 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
 
     fn lookup_op_method(
         &self,
-        lhs_ty: Ty<'tcx>,
+        lhs: (&'tcx hir::Expr<'tcx>, Ty<'tcx>),
```

Comment:
This can be
```suggestion
        (lhs_expr, lhs_ty): (&'tcx hir::Expr<'tcx>, Ty<'tcx>),
```
and avoid line 873.

Reply:
Fixed, thanks!